client/src/components/RichTextEditor.js

```diff
@@ -64,11 +68,71 @@ const RichTextEditor = ({
       editor.children = createSlateValue(value)
       editor.onChange()
     }
-  }, [editor, previousValue, readOnly, value])
+  }, [editor, previousValue, readOnly, disableFullSize, value])
 
   const renderElement = useCallback(props => <Element {...props} />, [])
   const renderLeaf = useCallback(props => <Leaf {...props} />, [])
 
+  const editableRef = useRef()
+  const toolbarRef = useRef()
+
+  useEffect(() => {
+    editableRef.current = ReactEditor.toDOMNode(editor, editor)
+  }, [editor])
+
+  const handleFullSizeMode = isFullSize => {
+    handleBannerHeight()
+    handleToolbarHeight()
+    setShowFullSize(isFullSize)
+  }
+  const handleBannerHeight = () => {
+    const topOffset =
+      document.getElementById("securityBannerContainer").getBoundingClientRect()
+        .bottom + "px"
+    document.documentElement.style.setProperty("--banner-height", topOffset)
```

Comment:
This is a bit ugly; maybe look into styled components for a cleaner way to accomplish this? See e.g. https://emotion.sh/docs/best-practices#advanced-css-variables-with-style